src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectBuildWrapper/help-propertiesContent.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
Now I get the reason. In such case you should reference a text with extra details by URL. It will prevent confusions of users